server/src/config.rs

```diff
@@ -0,0 +1,225 @@
+use crate::{PersistedDatabaseRules, ProvidedDatabaseRules};
+use async_trait::async_trait;
+use data_types::server_id::ServerId;
+use generated_types::database_rules::encode_persisted_database_rules;
+use generated_types::google::FieldViolation;
+use generated_types::influxdata::iox::management;
+use iox_object_store::IoxObjectStore;
+use object_store::ObjectStore;
+use snafu::{ensure, ResultExt, Snafu};
+use std::sync::Arc;
+use uuid::Uuid;
+
+/// A generic opaque error
+pub type StdError = Box<dyn std::error::Error + Send + Sync + 'static>;
```

Comment:
```suggestion
pub type BoxedError = Box<dyn std::error::Error + Send + Sync + 'static>;
```

I don't feel strongly, just I felt slightly mislead with the `std` prefix